ModificationResponse/ColorContributor.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+
+namespace Unity.Labs.SuperScience
+{
+    public class ColorContributor : MonoBehaviour
```

Comment:
A little comment here explaining what this script is for and pointing users to the modificationresponseexample window would be good